package.json

```diff
@@ -16,11 +16,12 @@
   },
   "homepage": "https://docs.pi-hole.net/",
   "scripts": {
-    "build": "mkdocs build --clean",
+    "build": "mkdocs build --clean --strict",
     "markdownlint": "markdownlint-cli2 \"**/*.md\" \"!**/node_modules/**\"",
-    "linkinator": "npm run build && linkinator site --recurse --silent --skip \"^(?!http://localhost)\"",
+    "linkinator": "linkinator site --recurse --silent --skip \"^(?!http://localhost)\"",
     "test": "npm run markdownlint && npm run linkinator",
-    "serve": "mkdocs serve --dev-addr 0.0.0.0:8000"
+    "serve": "mkdocs serve --dev-addr 0.0.0.0:8000",
+    "start": "npm run serve"
```

Comment:
> I also added a npm start script to start the server as a shortcut.

I'm a bit undecided about this. There is not much difference in `npm run start` vs `npm run serve`. Or is `npm run start` something 'common' one should have (best practice?)

Reply:
The difference is that one can do `npm start` and it works, while `serve` needs `npm run serve`. So, it's better to follow the standard here and make everyone's lives easier :)